vite.config.js

```diff
@@ -0,0 +1,58 @@
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue2'
+import EnvironmentPlugin from 'vite-plugin-environment'
+import ViteRequireContext from '@originjs/vite-plugin-require-context'
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require('path')
+
+const fs = require('fs')
+const packageJson = fs.readFileSync('./package.json')
+const appName = JSON.parse(packageJson).appName
+const appVersion = JSON.parse(packageJson).version
+const sbcName = JSON.parse(packageJson).sbcName
+const sbcVersion = JSON.parse(packageJson).dependencies['sbc-common-components']
+const aboutText1 = (appName && appVersion) ? `${appName} v${appVersion}` : ''
+const aboutText2 = (sbcName && sbcVersion) ? `${sbcName} v${sbcVersion}` : ''
+
+// https://vitejs.dev/config/
+export default defineConfig(() => {
+  return {
+    define: {
+      'import.meta.env.ABOUT_TEXT':
+        (aboutText1 && aboutText2) ? `"${aboutText1}<br>${aboutText2}"`
+          : aboutText1 ? `"${aboutText1}"`
+            : aboutText2 ? `"${aboutText2}"`
+              : ''
+    },
+    envPrefix: 'VUE_APP_', // Need to remove this after fixing vaults. Use import.meta.env with VUE_APP.
```

Comment:
Usually, if you want to `import.meta.env`, the environment variables must be prefixed with `VITE_`. This overrides that and makes `import.meta.env` work with `VUE_APP_` as before.

The reason why I didn't rename environment variables to have a prefix of `VITE_` is because of the ops code in one of the directories. That's how we usually populate our environment variables like the Genesys web message for example.

Reply:
Did you try renaming the env variables in vaults.env? If that didn't work then we'll need @pwei1018 to help with this (when he returns from vacation).

Reply:
Sev, can I please see what the variables look like in 1Password? You showed me the vaults once but I can't remember. Did they have the environment variables prefixed with VUE_APP?

Reply:
The variables in 1Password do not start with VUE_APP. That's a local renaming of the env keys. The variables in 1P are the last token on each line:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/0db513cf-f23f-485e-9f4c-f59f0ed0c4f6)

Eg, AUTH_API_URL

Reply:
I don't know if you can simply rename the env keys. I'm guessing there's a GCP step that uses vaults.env to look up the keys, assigns them env keys, and then bakes them into the app js bundle.

I _think_ that `.env` is our local way of having the keys without going through GCP.

Reply:
Sev, I've tried to change the environment variables prefix to VITE_ and test in the temporary URL. It doesn't work 😟 

Check this link: https://business-create-dev--pr-556-nilgmamu.web.app/registration-define-business?id=T9SwNW7ISG

The web messaging doesn't appear. Looks like the environment variable wasn't populated. I'm guessing there is some configuration needed to be done like you mentioned. I'll be reverting this change.

Reply:
And I guess you're not merging this until we figure it out. @seeker25 ?

Reply:
I think it should be fine for the time being Sev. When I have VUE_APP as the prefix, all environment variables get populated correctly. I can see the web messaging etc..

It's just the address complete variable that's not getting populated. Also, I'm merging now not to main, but to a new branch [17118-vite-test-branch](https://github.com/bcgov/business-create-ui/tree/17118-vite-test-branch)

Please let me know if that's OK with you Sev.

Reply:
Right, you have that config to override the prefix.

Ok to try merging to a branch and deploying it! Maybe post a message to let other devs know.